src/plugins/messenger/MessengerPreview/components/MessengerTextWithQuickReplies/MessengerTextWithQuickReplies.tsx

```diff
@@ -1,104 +1,112 @@
-import { IFBMRegularMessage } from '../../interfaces/Message.interface';
```

Comment:
please revert undesired styling changes. E.g.: This file now has tab stops of 2 spaces, we always use 4

Reply:
fixed